templates/telco/index.html

```diff
@@ -0,0 +1,242 @@
+            <div class="row u-equal-height u-vertically-center">
+              <div class="col-6">
+                <h1>
+                    The 5G mobile <br>subscriber experience
+                </h1>
+                <p class="p-heading--four">
+                    Delighting network customers with 5G apps
+                </p>
+              </div>
+              <div class="col-6 u-align--center u-hide--small">
+                    {{ image (
+                        url="https://assets.ubuntu.com/v1/55cff1c7-Anbox_Telco_5G.svg",
+                        alt="",
+                        width="152",
+                        height="195",
+                        hi_def=True,
+                        loading="auto"
+                        ) | safe
+                    }}
+              </div>
+            </div>
+        </section>
+
+        <section class="p-strip--light">
+            <div class="row u-equal-height u-vertically-center">
+                <div class="col-6">
+                    <p>
+                        5G lifts broadband and latency limitations in mobile networks. </br>
+                        However, building scalable infrastructure to deliver 5G apps to mobile subscribers is expensive. Anbox Cloud is a scalable platform to host 5G apps in telco RANs in the cloud and at the edge.
+                    </p>
+                    <p>
+                        Leading telco service providers are rolling out 5G networks to offer a new generation of broadband access to their subscribers. The virtually unlimited bandwidth and latency of 5G unlock new possibilities. Telcos that create innovative value-added services will be winners in the 5G race.
+                    </p>
+                    <p>
+                        With 5G, elastic computing in the cloud becomes available to mobile users at zero latency. However, mobile handset capabilities are restricted and will remain a bottleneck. Offloading compute and storage  to the cloud via 5G networks could be a way to circumvent these limitations.
+                    </p>
+                    <p>
+                        <a href="">
+                            Learn about value-added telco services with Anbox Cloud >
+                        </a>
+                    </p>
+                </div>
+                <div class="col-6 u-align--center u-hide--small">
+                    {{ image (
+                        url="https://assets.ubuntu.com/v1/d5de891a-Acceleration.svg",
+                        alt="",
+                        width="287",
+                        height="160",
+                        hi_def=True,
+                        loading="auto"
+                        ) | safe
+                    }}
+                </div>
+            </div>
+        </section>
+
+        <section class="p-strip">
+            <div class="row u-vertically-center">
+                <div class="row col-7">
```

Comment:
I think `col-7` should be a child of `row` not in the same element. Try removing the `col-7` here and replacing the `col-9` with it on line 73. (as `col-9` + `col-5` is too many)